spec/linter-pep257-spec.coffee

```diff
@@ -0,0 +1,66 @@
+LinterPEP257 = require '../lib/init'
+path = require 'path'
+
+goodPath = path.join(__dirname, 'fixtures', 'good.py')
+badPath = path.join(__dirname, 'fixtures', 'bad.py')
+emptyPath = path.join(__dirname, 'fixtures', 'empty.py')
+
+describe "starts everything up", ->
+  lint = require('../lib/init').provideLinter().lint
+  beforeEach ->
+    waitsForPromise ->
+      atom.packages.activatePackage('linter-pep257')
+    waitsForPromise ->
```

Comment:
We don't have to make it a level deeper so this looks kinda good, but `await`s are even better like we do in `linter-eslint` specs